controllers/assignments.py

```diff
@@ -529,13 +528,13 @@ def _autograde_one_q(course_name, assignment_id, sid, qname, points, deadline=No
     question = db(db.questions.name == qname).select().first()
 
     # dispatch on grading_type; if none specified, can't autograde
-    if question.autograde == 'unittest':
+    if autograde == 'unittest':
```

Comment:
I wasn't aware of that grading path. I don't think I implemented autograde for anything other than assignments. What do you have in mind?

Note: we discussed last week and decided the decision about whether particular questions should be autograded should be a property of the assignment_question, not the question.